src/giskardpy/tree/behaviors/ros_msg_to_goal.py

```diff
@@ -58,7 +58,10 @@ def update(self):
             # traceback.print_exc()
             return Status.SUCCESS
         if self.god_map.get_data(identifier.collision_checker) != CollisionCheckerLib.none:
-            self.parse_collision_entries(move_cmd.collisions)
+            try:
+                self.parse_collision_entries(move_cmd.collisions)
+            except ConstraintInitalizationException:
```

Comment:
why are you catching this exception?

Reply:
I removed this now as this was not an intended to be pushed